examples/simple-form/src/SimpleForm.state.js

```diff
@@ -44,6 +46,14 @@ const formConfig = {
       validateWhen(matchesRegex("^meow$"), matchesRegex("^cat$"), "animal"),
       validateWhen(matchesRegex("^moo$"), matchesRegex("^cow$"), "animal")
     ]
+  },
+  numberOfDogs: {
+    validators: [validateSum(numberLessThan(5), ["numberOfCats"])],
```

Comment:
I know this is an old PR, but it might be better to just have validateSum renamed to validateSumIsLessThan since this wouldn't work with just any generic validator. Not a huge deal, but maybe we can come back to it next time we add a validator